### ISSUE TITLE
Sigrok support and i2s support

### ISSUE THREAD

**Issue by abidbodal** [embeddedartists/labtool#14]
Hi,

Does the labtool support i2s? What would it take to add a protocol? Is there any documentation on this?

Does the labtool have sigrok support? If not, what do you think it would take to add this support?

**Comment by embeddedartists** [comment]
No, there isn't any analyzer for I2S and no support for sigrok. Please note that LabTool is no a non-stock product.
htpts://www.embeddedartists.com/products/labtool/#orderinginfo
